test/Console/AzureExtension.TestConsole.csproj

```diff
@@ -44,7 +44,7 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="4.56.0" />
     <PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Version="4.56.0" />
     <PackageReference Include="Microsoft.Toolkit.Uwp.Notifications" Version="7.1.3" />
-    <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.100.254" />
+    <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.100.323" />

```

Comment:
See comment about SDK changes above and what we need to do

Reply:
Realized after but Huzaifa already has this PR targeted to a forked branch, resolving this.